teacher.rb

```diff
@@ -0,0 +1,20 @@
+require './person'
+
+class Teacher < Person
+  # getter/setter method
+  attr_accessor :specialization
+
+  # constructor method
+  def initialize(specialization)
```

Comment:
Hello! 🌟 In order to ensure that the Teacher class's initialize method accepts age, name (with a default value of "unknown"), and parent_permission (with a default value of true), it's important to pass these parameters to both the Teacher class's initialize method and the super call within the Teacher class. By doing so, you're ensuring that the Teacher class inherits all the attributes from its parent class, Person. 🚀📚